app/views/static_pages/_connect.html.slim

```diff
@@ -26,7 +26,7 @@
               #carousel-connect.carousel.slide data-ride="carousel"
                 .row
                   .carousel-inner.home-connect__feature-carousel role="listbox"
-                    - Network.includes(:users).limit(9).each_with_index do |network, index|
+                    - networks.each_with_index do |network, index|
```

Comment:
Curious why this in here? Didn't we already merge this feature?

Reply:
I just moved the query to the controller. it is part of this PR. See StaticPagesController#home for further information.

Reply:
You are right. Sorry.